info.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "malwareAttackResponse",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Malware Attack Response",
+    "dependencies": [
+        {
+            "name": "fortiSOAREssentials",
+            "type": "solutionpack",
+            "label": "FortiSOAR Essentials",
+            "version": "1.0.0",
+            "minVersion": null
+        },
+        {
+            "name": "socSimulator",
+            "type": "solutionpack",
+            "label": "SOC Simulator",
+            "version": "1.0.0",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": [],
+    "category": [],
+    "tags": [],
+    "certified": "true",
+    "publisher": "Fortinet",
+    "supportInfo": "support@fortinet.com",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-malware-attack-response",
+    "description": "This Solution Pack deals with containment and investigation procedures for Malware detected on assets.",
+    "fsrMinCompatibility": "7.2.0",
+    "date": "2022-02-16T17:50:38+00:00",
+    "exported_from": "master.fortisoar.com",
+    "exported_by": "CS Admin",
+    "contents": {
+        "modules": {
+            "scenario": []
+        },
+        "viewTemplates": {
+            "scenario": [
+                "list",
+                "detail",
+                "form"
+            ]
+        },
+        "playbooks": [
+            {
+                "name": "04 - Use Case - Malware Attack Response"
+            }
+        ],
+        "connectors": [
+            {
+                "name": "smtp",
+                "version": "2.4.2"
```

Comment:
This is system connector and will always be installed, right? what if SMTP connector version changes with every release and we don't update it here?

Reply:
These files are auto-created .. when it will be installed, it will take the latest version